.github/workflows/lint.yml

```diff
@@ -37,7 +37,7 @@ jobs:
 
     - name: Lua Check
       if: steps.changed-files.outputs.any_changed == 'true'
-      uses: Kong/public-shared-actions/code-check-actions/lua-lint@33449c46c6766a3d3c8f167cc383381225862b36
+      uses: Kong/public-shared-actions/code-check-actions/lua-lint@ac8939f0382827fbb43ce4e0028066a5ea4db01d #v4.1.2
```

Comment:
Could it be another PR?

Reply:
If this PR is hard to review originally, i would like to bring unrelated modification out of this PR. But this PR is enough simple, so we could merge two thing(function + other stuff like CI version bump) into one. Otherwise it will bring a lot of unnecessary work. I don't like the experience of opening multiple PRs.

Let machines serve us, not the other way around. People's feelings matter more than the rigidity of machines, and I don't think this PR will affect future work. 😃